Packages/com.unity.inputsystem/InputSystem/Devices/Touchscreen.cs

```diff
@@ -157,10 +157,12 @@ public struct TouchState : IInputStateTypeInfo
         [FieldOffset(33)]
         public byte tapCount;
 
-        // Not currently used, but still needed in this struct for padding,
-        // as il2cpp does not implement FieldOffset.
+        /// <summary>
+        /// The index of the display that was touched.
+        /// </summary>
+        [InputControl(name = "displayIndex", displayName = "Display Index", layout = "Integer")]
         [FieldOffset(34)]
-        byte displayIndex;
+        public byte displayIndex;
```

Comment:
This is more of a side not than feedback on this PR, but I see this is not coming from this PR but its unfortunate to see types of displayIndex aren't aligned here, e.g. byte here and ushort in another place. I do not see that narrowing integer types is a a problem since systems rarely support a high number of logical screens anyway, but unfortunate inconsistency.